go/enclave/rpc/EstimateGas.go

```diff
@@ -148,44 +153,45 @@ func (rpc *EncryptionManager) doEstimateGas(args *gethapi.TransactionArgs, blkNu
 		*/
 		hi = rpc.config.GasLocalExecutionCapFlag
 	}
-	// Normalize the max fee per gas the call is willing to spend.
-	var feeCap *big.Int
-	if args.GasPrice != nil && (args.MaxFeePerGas != nil || args.MaxPriorityFeePerGas != nil) {
-		return 0, errors.New("both gasPrice and (maxFeePerGas or maxPriorityFeePerGas) specified")
-	} else if args.GasPrice != nil {
-		feeCap = args.GasPrice.ToInt()
-	} else if args.MaxFeePerGas != nil {
-		feeCap = args.MaxFeePerGas.ToInt()
-	} else {
-		feeCap = gethcommon.Big0
-	}
-	// Recap the highest gas limit with account's available balance.
-	if feeCap.BitLen() != 0 { //nolint:nestif
-		balance, err := rpc.chain.GetBalanceAtBlock(*args.From, blkNumber)
-		if err != nil {
-			return 0, fmt.Errorf("unable to fetch account balance - %w", err)
+	/*
+		// Normalize the max fee per gas the call is willing to spend.
```

Comment:
Worth adding a comment why you're commenting this out if you don't think it can be deleted, maybe a todo to help someone revisit it?

Reply:
it was for debugging. will change back.